### PR TITLE
Account-wise active stake cache

### PR DESCRIPTION
**Pull request by DamjanOstrelic**
- cache setup
- account history endpoint

### CODE REVIEW

**Review by rdlrt — COMMENTED** [review]
Need to add update to specs

File: files/grest/rpc/01_cached_tables/active_stake_cache.sql [hunk 1]
```diff
@@ -162,10 +171,43 @@ $$
...
+    INSERT INTO GREST.ACCOUNT_ACTIVE_STAKE_CACHE
+      SELECT
+        STAKE_ADDRESS.VIEW AS STAKE_ADDRESS,
+        POOL_HASH.VIEW AS POOL_ID,
+        EPOCH_STAKE.EPOCH_NO AS EPOCH_NO,
+        SUM(EPOCH_STAKE.AMOUNT) AS AMOUNT
+      FROM
+        EPOCH_STAKE
+        INNER JOIN POOL_HASH ON POOL_HASH.ID = EPOCH_STAKE.POOL_ID
+        INNER JOIN STAKE_ADDRESS ON STAKE_ADDRESS.ID = EPOCH_STAKE.ADDR_ID
```
Comment (rdlrt) [inline files/grest/rpc/01_cached_tables/active_stake_cache.sql:190]: For consistency (tho we might wanna check if we missed to refer elsewhere), mentioning schema..

Comment (DamjanOstrelic) [reply files/grest/rpc/01_cached_tables/active_stake_cache.sql:190]: > Need to add update to specs

yep, just wanted to check first if that's the endpoint we want and whether we want to use this cache in some existing endpoints.

**Comment by DamjanOstrelic** [comment]
> Need to add update to specs

yep, just wanted to check first if that's the endpoint we want and whether we want to use this cache in some existing endpoints.

**Review by rdlrt — APPROVED** [review]

**Pull request merged**